src/jaxsim/api/frame.py

```diff
@@ -49,19 +60,18 @@ def name_to_idx(model: js.model.JaxSimModel, *, frame_name: str) -> jtp.Int:
         The index of the frame.
     """
 
-    if frame_name in model.kin_dyn_parameters.frame_parameters.name:
-        return (
-            jnp.array(
-                np.argwhere(
-                    np.array(model.kin_dyn_parameters.frame_parameters.name)
-                    == frame_name
-                )
-            )
-            .squeeze()
-            .astype(int)
-        ) + model.number_of_links()
+    if frame_name not in model.kin_dyn_parameters.frame_parameters.name:
+        raise ValueError(f"Frame '{frame_name}' not found in the model.")
 
-    return jnp.array(-1).astype(int)
+    return (
+        jnp.array(
+            np.argwhere(
+                np.array(model.kin_dyn_parameters.frame_parameters.name) == frame_name
+            )
+        )
+        .astype(int)
+        .squeeze()
+    ) + model.number_of_links()
```

Comment:
We can use the list functions directly. Be aware that `numpy` is no more used
```suggestion
    return (
        jnp.array(
            model.kin_dyn_parameters.frame_parameters.name.index(frame_name)
        ).squeeze()
    ) + model.number_of_links()
```

Reply:
I'm not sure if we can index attributes (neither if static) within a jit-compiled function. I can give it a try.